src/audio/component.c

```diff
@@ -196,24 +196,6 @@ int comp_get_copy_limits(struct comp_dev *dev, struct comp_copy_limits *cl)
 	cl->sink = list_first_item(&dev->bsink_list, struct comp_buffer,
 				   source_list);
 
-	/* check for underrun */
```

Comment:
Wouldn't this be better under a Kconfig option for DEBUG_XRUN, that way it could be easily switched back on to debug any pipeline / component / topology issue.

Reply:
@lgirdwood I have no problem with that, but I'm afraid of users enabling this debug option for pipelines, where it is expected for components to sometimes not produce/consume data. It will generate false positives. Also currently if we are going to call produce/consume with 0 bytes it will generate trace, so it's not like we cannot debug this kind of issue.

Reply:
@tlauda the false negatives are a problem but we do need something whereby we can track the amount of data moving between components and pipelines. I assume this information would come from probes ?